charts/geth/values.yaml

```diff
@@ -150,7 +150,7 @@ readinessProbe:
 httpPort: 8545
 
 # -- WS Port
-wsPort: 8546
+wsPort: 8545
```

Comment:
I think the wsPort isn't being used anywhere? Missing something in `defaultCommandTemplate` maybe?

Reply:
Oh right. You just changed the port to be the same as the httpPort by default. I didn't know that we could reuse the same port on both flags, that's great 👍 

Just tried it out myself `docker run --rm -it ethereum/client-go --http --http.port=8545 --ws --ws.port=8545 `